### PR TITLE
[vtctld] vtctldclient command pkg

### PR DESCRIPTION
**Pull request by ajm188**
## Description

This moves the command definitions and logic to a subpackage, to allow us to more easily break up commands into files logically, rather than shoving everything into a single "commands" file.

For testing, I validated I could still compile and run an rpc, which to me is enough confirmation this is a no-op.

## Related Issue(s)

- 

## Checklist
- [ ] Should this PR be backported? **No**
- [ ] Tests were added or are not required **No**
- [x] Documentation was added or is not required

## Impacted Areas in Vitess
Components that this PR will affect:

- [ ]  Query Serving
- [ ]  VReplication
- [ ]  Cluster Management
- [ ]  Build 
- [ ]  VTAdmin

### CODE REVIEW

File: go/cmd/vtctldclient/internal/command/doc.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Package command contains the commands used by vtctldclient. It is intended only
```
Comment (doeg) [inline go/cmd/vtctldclient/internal/command/doc.go:18]: ❤️ ❤️ ❤️

Comment (doeg) [reply go/cmd/vtctldclient/internal/command/doc.go:18]: Seriously, these docs are so so good.

**Review by doeg — APPROVED** [review]
So nice and tidy! ✨

File: go/cmd/vtctldclient/internal/command/doc.go (hunk 1 above)
Comment (deepthi) [reply go/cmd/vtctldclient/internal/command/doc.go:18]: +100

**Pull request merged**